hack/local-up-cluster.sh

```diff
@@ -237,7 +237,6 @@ CPU_CFS_QUOTA=${CPU_CFS_QUOTA:-true}
 ENABLE_HOSTPATH_PROVISIONER=${ENABLE_HOSTPATH_PROVISIONER:-"false"}
 CLAIM_BINDER_SYNC_PERIOD=${CLAIM_BINDER_SYNC_PERIOD:-"15s"} # current k8s default
 ENABLE_CONTROLLER_ATTACH_DETACH=${ENABLE_CONTROLLER_ATTACH_DETACH:-"true"} # current default
-KEEP_TERMINATED_POD_VOLUMES=${KEEP_TERMINATED_POD_VOLUMES:-"true"}
```

Comment:
also update `  --keep-terminated-pod-volumes="${KEEP_TERMINATED_POD_VOLUMES}"` below

Reply:
sorry, why do I need to update below? wouldn't that use this variable declared here?

Reply:
grr, now I see. thanks